src/Components/Aspire.Microsoft.EntityFrameworkCore.Cosmos/AspireAzureEFCoreCosmosExtensions.cs

```diff
@@ -23,6 +23,47 @@ public static class AspireAzureEFCoreCosmosExtensions
     private const string DefaultConfigSectionName = "Aspire:Microsoft:EntityFrameworkCore:Cosmos";
     private const DynamicallyAccessedMemberTypes RequiredByEF = DynamicallyAccessedMemberTypes.PublicConstructors | DynamicallyAccessedMemberTypes.NonPublicConstructors | DynamicallyAccessedMemberTypes.PublicProperties;
 
+    /// <summary>
+    /// Registers the given <see cref="DbContext" /> as a service in the services provided by the <paramref name="builder"/>.
+    /// Derives the name of the database from the connection string.
```

Comment:
Should it also derive the name of the container (if specified)? Is that even possible from the outside of the DbContext?

cc @AndriySvyryd - is it possible to set the CosmosDB container name outside of the DbContext?

Reply:
By default, the container name is derived from the context type name. As @captainsafia mentioned in the description, the default container name can be set using a convention, but it requires some additional infrastructure.